docs/RFCS/explain_debug.md

```diff
@@ -0,0 +1,119 @@
+- Feature Name: explain_debug
+- Status: in-progress
+- Start Date: 2015-08-10
+- RFC PR: #2047
+- Cockroach Issue:
+
+# Summary
+
+Add support for an `EXPLAIN DEBUG` statement. The `EXPLAIN DEBUG` statement
+will cause simple (single table or index) select statements to display the
+underlying key-value pairs that were accessed:
```

Comment:
That's a good point. I'm going to leave this as-is for now, but extending the debugging facilities should be on the top of our mind. Changing the syntax from `EXPLAIN DEBUG` to `DEBUG` or `DEBUG KEYS` should be trivial.